app/admin/models/Orders_model.php

```diff
@@ -154,9 +158,25 @@ public function scopeListFrontEnd($query, $options = [])
             }
         }
 
+        $search = trim($search);
+        if (strlen($search)) {
+            $query->search($search, $searchableFields);
+        }
+
+        if (($startDateTime = array_get($dateTimeFilter, 'orderDateTime.start', false)) AND ($endDateTime = array_get($dateTimeFilter, 'orderDateTime.end', false))) {
+            $query = $this->scopeWhereBetweenOrderDateTime($query, Carbon::parse($startDateTime)->format('Y-m-d H:i:s'), Carbon::parse($endDateTime)->format('Y-m-d H:i:s'));
+        }
```

Comment:
Lastly, parenthesis in the `if` statement has no effect so not need. `$startDateTime = array_get($dateTimeFilter, 'orderDateTime.start', false) AND $endDateTime = array_get($dateTimeFilter, 'orderDateTime.end', false)` will work.

Reply:
Changed!